.php_cs.dist

```diff
@@ -48,6 +48,9 @@ return PhpCsFixer\Config::create()
             'header' => $header,
             'location' => 'after_open',
         ],
+        'native_function_invocation' => [
```

Comment:
Does this merge with a rule or does it replace the rule entry? Because It's already present in the rule @Symfony:risky

Reply:
Yes, my bad, recent changes have misled me: 
* https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4043
* https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4220